src/Connection/Http2Connection.php

```diff
@@ -243,6 +243,11 @@ public function isBusy(): bool
         return $this->remainingStreams <= 0 || $this->socket->isClosed();
     }
 
+    public function getRemainingTime(): ?int
```

Comment:
I don't believe there is sensible way to implement this with HTTP/2, so I do not think it should be part of the `Connection` interface.